source/system-administrators/activities/autoclustering-with-studio.rst

```diff
@@ -47,170 +63,235 @@ Requirements
+   ``studio-config-overrides.yaml`` file.
+
+   .. code-block:: yaml
+      :caption: *bin/apache-tomcat/shared/classes/crafter/studio/extension/studio-config-override.yaml*
+
+      # Cluster member registration, this registers *this* server into the pool
+      # Cluster node registration data, remember to uncomment the next line
+      # studio.clustering.node.registration:
+      #  This server's local address (reachable to other cluster members). You can also specify a different port by 
+      #  attaching :PORT to the adddress (e.g 192.168.1.200:2222)
+      #  localAddress: ${env:CLUSTER_NODE_ADDRESS}
+      #  Authentication type to access this server's local repository
+      #  possible values
+      #   - none (no authentication needed)
+      #   - basic (username/password authentication)
+      #   - key (ssh authentication)
+      #  authenticationType: none
+      #  Username to access this server's local repository
+      #  username: user
+      #  Password to access this server's local repository
+      #  password: SuperSecurePassword
+      #  Private key to access this server's local repository (multiline string)
+      #  privateKey: |
+      #    -----BEGIN PRIVATE KEY-----
+      #    privateKey
+      #    -----END PRIVATE KEY-----
 
-   where:
+   |
 
-   - **studio.clustering.node.registration.localAddress**: this server's local address (address reachable to other cluster members)
-   - **studio.clustering.node.registration.authenticationType**: authentication type to access this server's local repository
+   Uncomment and leave the value of  **studio.clustering.node.registration.localAddress** as 
+   ``${env:CLUSTER_NODE_ADDRESS}`` (you will configure the node address in a later step), then configure the 
+   repository authentication:
+   
+   - **studio.clustering.node.registration.authenticationType**: authentication type to access this server's local 
+     repository
    - **studio.clustering.node.registration.username**: username to access this server's local repository
    - **studio.clustering.node.registration.password**: password to access this server's local repository
-   - **studio.clustering.node.registration.privateKey**: private key to access this server's local repository (multiline string) when using ``key`` as authentication type to access this server's local repository
+   - **studio.clustering.node.registration.privateKey**: private key to access this server's local repository 
+     (multiline string) when  using ``key`` as authentication type to access this server's local repository
 
    |
 
-   Modify the values as shown above in the clustering section of your ``studio-config-overrides.yaml`` file with values from your setup and save the file.  You can also change the clustering section from the |mainMenu| **Main Menu** in Studio under ``Global Config``
+   .. note:: 
+      You can used the node's default SSH keys, located in ``~/.ssh/id_rsa`` and ``~/.ssh/id_rsa.pub``, if you set
```

Comment:
Typo? `use` instead of `used` so it goes like:
```
You can use the node's default SSH keys ...
```

Reply:
@alhambrav typo fixed